src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead tr');
+
+thead.addEventListener('click', (event) => {
+  const list = document.querySelector('tbody');
+  const rows = [...document.querySelectorAll('tbody tr')];
+  const point = event.target.closest('th');
+  const index = [...thead.children].indexOf(point);
+
+  if (index === 0 || index === 1) {
+    rows.sort((a, b) =>
+      a.children[index].innerHTML.localeCompare(b.children[index].innerHTML)
+    );
+  }
+
+  if (index === 2 || index === 3) {
+    rows.sort((a, b) =>
+      a.children[index].innerHTML.replace(/\D/g, '')
+      - b.children[index].innerHTML.replace(/\D/g, '')
```

Comment:
```suggestion
      +a.children[index].innerHTML.replace(/\D/g, '')
      - +b.children[index].innerHTML.replace(/\D/g, '')
```
show that this is number